### DISCUSSION TITLE
Module per compression format

### DISCUSSION THREAD

**Discussion by coxley** [klauspost/compress#895] [Ideas/Features]
Howdy!

A small nit: pulling this library in on slow networks can take a long time. I wonder if it would be improved if each format had it's own `go.mod`.

**Comment by klauspost** [comment]
I am looking into moving most test data into a separate repo. At some point go devs were promosing that go downloads would exclude testdata, but that never seem to have materialized.

I don't think "sub-modules" is the correct approach.

**Reply by coxley** [reply]
That makes a lot of sense. Sorry for bugging about something you're already on top of. :)
